vulcan/hebe/data/_homework.py

```diff
@@ -41,7 +41,7 @@ class Homework(Serializable):
     date_created: DateTime = ChildField(DateTime, key="DateCreated")
     creator: Teacher = ChildField(Teacher, key="Creator")
     subject: Subject = ChildField(Subject, key="Subject")
-    attachments: list = SequenceField(list, key="Attachments")
+    attachments: list = SequenceField(Attachment, key="Attachments")
```

Comment:
```suggestion
    attachments: list[Attachment] = SequenceField(Attachment, key="Attachments", repr=True)
```

Reply:
What does this line do?

Reply:
It allows you to obtain a clear effect and is useful in documentation.

Reply:
So docs, should look like this?
`:var List[Attachment]  ~.attachments: attachments added to homework`

Reply:
Yes but you should also update docs in docs folder

Reply:
I don't think so; there are no per-class docs in any other place than the docstring inside the class definition. Changing `list` to `List` (from `typing`) improves compatibility with docs (e.g. links to the target class) and IDE, which may use the class as a type hint.